src/components/container/RelativeActivity/RelativeActivity.tsx

```diff
@@ -0,0 +1,111 @@
+import { faChevronUp } from "@fortawesome/free-solid-svg-icons";
+import { startOfDay } from "date-fns";
+import React, { useContext } from "react";
+import { useState } from "react";
+import { FontAwesomeSvgIcon } from "react-fontawesome-svg-icon";
+import getDayKey from "../../../helpers/get-day-key";
+import { getDailyDataTypeDefinition } from "../../../helpers/query-daily-data";
+import { ActivityMeter, CardTitle, DateRangeContext, } from "../../presentational";
+import "./RelativeActivity.css"
+import language from "../../../helpers/language";
+import { useInitializeView } from "../../../helpers/Initialization";
+import { RelativeActivityContext } from "../RelativeActivityDayCoordinator/RelativeActivityDayCoordinator";
+import { RelativeActivityDataType, RelativeActivityQueryResult } from "../../../helpers";
+import { queryRelativeActivity } from "../../../helpers/relative-activity";
+
+export interface RelativeActivityProps {
+    dataTypes?: RelativeActivityDataType[];
+    useContext?: boolean;
+    previewState?: "Default";
+    title?: string;
+    innerRef?: React.Ref<HTMLDivElement>
+    date?: Date;
+}
+
+export default function (props: RelativeActivityProps) {
+    let [results, setResults] = useState<{ [key: string]: RelativeActivityQueryResult } | undefined>(undefined);
+    let relativeActivityContext = useContext(RelativeActivityContext);
+
+    let dataTypes: RelativeActivityDataType[] = props.dataTypes || [];
+    if (props.useContext && relativeActivityContext?.dataTypes) {
```

Comment:
Under what circumstances would you wrap this component with a relative activity context, but *not* want to use it?

Reply:
I think if you wanted to expand/change the data types that are shown on a detail view.  Like the RelativeActivityCoordinator could have "sleep" as one of the bars, when you click a day it shows a RelativeActivity component with "Sleep" but then a second one with REM, Deep, Light sleep etc.